### ISSUE TITLE
doc: how many whenever blocks (within react blocks) can run concurrently in a full program?

### ISSUE THREAD

**Issue by cxw42** [Raku/doc#3123]
## The problem

Sterling Hanenkamp's [Advent calendar entry on `react`](https://zostay.com/archive/2019/12/07/react-blocks.html) says:

> never will any code within [a `react`] block be run concurrently

[The docs] (https://docs.raku.org/language/concurrency#index-entry-whenever) say:

> only one whenever block will be executed at any time.

Does execution of a`whenever` block within a`react` block exclude concurrent execution of all other `whenever` blocks only in the same `react`, in the same thread, in the whole program, or in some other scope?  I am hoping it's the first of those, since that would mean each `react` block could serve as a monitor independently of any other `react` blocks.

## Suggestions

If `react` blocks are monitors, document them as such.  If not, clarify the scope of mutual exclusion they provide.

**Comment by jnthn** [comment]
> Does execution of awhenever block within areact block exclude concurrent execution of all other whenever blocks only in the same react

Yes, each time a `react` is encountered in a program, it can be considered a new instance (or, put another way, to start a new actor). The subscriptions established by the `whenever` are thus associated with that instance. One may therefore `start` numerous `react` blocks, which will be able to process messages concurrently; the concurrency control applies to a single instance.

> If react blocks are monitors, document them as such. 

I don't think "monitor" is a good term to use here, since those tend to be more used in a blocking concurrency model. I guess "actor" is a bit closer, but it's probably best to just explain what the semantics are rather than what they're like.
